app/apollo/models/resource.schema.js

```diff
@@ -15,10 +15,11 @@
  */
 
 const mongoose = require('mongoose');
+const ObjectId = require('mongoose').Types.ObjectId;
 
 const ResourceSchema = new mongoose.Schema({
   _id: {
-    type: String,
+    type: ObjectId,
```

Comment:
Wont forcing this to be an objectid cause problems later? Because we'll need to update all the foreign keys in other tables to be objectids too, and objectids are also effort to compare.

Reply:
we probably do that later. this is a quick patch to get staging working.